src/main/java/org/reactome/qa/doisuggester/Main.java

```diff
@@ -50,213 +56,201 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 			Map<GKInstance, Set<Map<GKInstance, GKInstance>>> pathway2Reactions2NewIEs = new HashMap<>();
 
 			// Iterate through all ReactionlikeEvents (RlEs) in the 'current' test_reactome
-			for (GKInstance currentRLE : (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent))
+			Collection<GKInstance> RLEs = (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent);
+			System.out.println(RLEs.size() + " Reaction-like events will be checked.");
```

Comment:
Very minor thing, but ReactionlikeEvent is usually one word without a hyphen.

Reply:
When programming, you can't put a hyphen in a variable name, but I think the more grammatically correct way to write it is with a hyphen.

https://www.proofreadnow.com/blog/master-prefixes-and-suffixes-with-hyphens

Reply:
True, but this isn't the way it is written or referred to in the Reactome data model and in other code.

Reply:
That doesn't mean it's grammatically correct.

Reply:
Yes, I am just reluctant to start diverging into other ways of referring to classes.  This is really minor - I won't insist if you would prefer to leave it as is.

Reply:
It will remain.